flink-runtime/src/test/java/org/apache/flink/runtime/testutils/InMemoryReporterRule.java

```diff
@@ -18,8 +18,13 @@
 package org.apache.flink.runtime.testutils;
 
 import org.apache.flink.annotation.Experimental;
+import org.apache.flink.metrics.Metric;
+import org.apache.flink.metrics.MetricGroup;
+import org.apache.flink.runtime.testutils.InMemoryReporter.CaptureMode;
 import org.apache.flink.util.ExternalResource;
 
+import static org.apache.flink.util.Preconditions.checkNotNull;
+
 /**
  * A JUnit rule that encapsulates {@link InMemoryReporter} properly with test cases.
```

Comment:
I think we have to update the docstring slightly because without using this rule the `InMemoryReporter`  of the minicluster does not capture anything anymore.